sphinx/util/inspect.py

```diff
@@ -614,6 +614,17 @@ def _should_unwrap(subject: Callable) -> bool:
     return False
 
 
+def _fix_type_alias_forward_ref(annotation: Any) -> Any:
```

Comment:
How about `expand_type_alias_forward_ref()` instead?

Additonally, it would be better to add a docstring to this.